ddl/db_integration_test.go

```diff
@@ -2225,7 +2225,7 @@ func (s *testIntegrationSuite7) TestAddExpressionIndex(c *C) {
 }
 
 func (s *testIntegrationSuite7) TestCreateExpressionIndexError(c *C) {
-	defer config.RestoreFunc()
+	defer config.RestoreFunc()()
```

Comment:
It is easy to make mistake to call a function of function, is there anyway we can do better?

Reply:
Yeah, it is easy to go wrong. How about this:

```go
restoreFn := config.UpdateGlobal(func(conf *config.Config) {
	conf.Experimental.AllowsExpressionIndex = true
})
defer restoreFn()
```